README.md

```diff
@@ -93,6 +93,9 @@ reported a the [GibHub repository](https://github.com/dotnet-project-file-analyz
 * [**Proj0039** Treat all warnings as errors is considered a bad practice](rules/Proj0039.md)
 * [**Proj0040** Run NuGet security audits on transitive dependencies too](rules/Proj0040.md)
 * [**Proj0041** NuGet security audits should report on moderate issues at minimum](rules/Proj0041.md)
+* [**Proj0042** Use lock files](rules/Proj0042.md)
+* [**Proj0043** Enable &lt;RestoreLockedMode&gt; when &lt;ContinuousIntegrationBuild&gt; is enabled](rules/Proj0043.md)
```

Comment:
It feels more logical to me if rule 43 and rule 44 are switched, because having to enable RestoreLockedMode is a 'logical result' of enabling ContiuousIntegrationBuild.